### PR TITLE
feat: adjust KB homepage, clean resources and public folder [BAU-830]

### PR DESCRIPTION
**Pull request by bgutsol**

### CODE REVIEW

**Review by mshaaban0 — APPROVED** [review]
"This website is built with......" seems like dummy content, let's remove that otherwise looks good.
Screenshot 2022-06-24 at 17 32 57

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (denkristoffer) [inline .prettierrc:1]: Is this necessary?

Comment (bgutsol) [reply .prettierrc]: no, will remove

**Review by denkristoffer — APPROVED** [review]
Thanks, did you test the build with this?

**Comment by bgutsol** [comment]
> Thanks, did you test the build with this?

not yet, but gonna test now

**Comment by bgutsol** [comment]
> Thanks, did you test the build with this?

tested, now the KB starter flow is working

**Pull request merged**